paymentsheet/src/main/java/com/stripe/android/link/ui/SecondaryButton.kt

```diff
@@ -25,7 +25,7 @@ internal fun SecondaryButton(
         onClick = onClick,
         modifier = modifier
             .fillMaxWidth()
-            .height(PrimaryButtonHeight),
+            .height(56.dp),
```

Comment:
👀 If this were intended to be side-by-side with `PrimaryButton`, it should use primary button height. But I don't think that's the case.